### PR TITLE
README.md edits

### PR DESCRIPTION
**Pull request by elenalape**
**WIP -- DO NOT MERGE**

## What does this PR change?
* Change referencing to OpenCost from cost-model

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,83 +1,75 @@
...
+# OpenCost — your favorite open source cost monitoring tool for Kubernetes
 
-![Kubecost allocation UI](/allocation-drilldown.gif)
+OpenCost models give teams visibility into current and historical Kubernetes spend and resource allocation. These models provide cost transparency in Kubernetes environments that support multiple applications, teams, departments, etc.
+
+OpenCost is developed by [Kubecost](https://kubecost.com).
+
+![OpenCost allocation UI](/allocation-drilldown.gif)
+
+To see more on the functionality of OpenCost, as well as the full Kubecost product, please visit the [features page](https://kubecost.com/#features) on our website.
```
Comment (dwbrown2) [inline README.md]: Link to OpenCost site here?

**Comment by dwbrown2** [comment]
@AjayTripathy would you be able to review quickly too?

File: README.md [hunk 2]
```diff
@@ -1,83 +1,75 @@
-## Kubecost
+<img src="./opencost-header.png"/>
 
-Kubecost models give teams visibility into current and historical Kubernetes spend and resource allocation. These models  provide cost transparency in Kubernetes environments that support multiple applications, teams, departments, etc.
+# OpenCost — your favorite open source cost monitoring tool for Kubernetes
 
-![Kubecost allocation UI](/allocation-drilldown.gif)
+OpenCost models give teams visibility into current and historical Kubernetes spend and resource allocation. These models provide cost transparency in Kubernetes environments that support multiple applications, teams, departments, etc.
+
+OpenCost is developed by [Kubecost](https://kubecost.com).
```
Comment (dwbrown2) [inline README.md]: Should we link to our intentions here? https://github.com/kubecost/cost-model/issues/1224

**Review by dwbrown2 — COMMENTED** [review]
Looks really good! Just a couple minor comments.

File: README.md (hunk 1 above)
Comment (elenalape) [reply README.md]: done

**Review by teevans — APPROVED** [review]
LGTM

**Pull request merged**